src/index.html

```diff
@@ -6,13 +6,66 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Product cards</title>
+    <title>Product card</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <section
+      class="product-card"
+      data-qa="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+        class="product-card__img"
+      />
+
+      <div class="product-card__details">
+        <h3 class="product-card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+        <p class="product-card__sku">Product code: 195434</p>
```

Comment:
it will be more readable with 1 empty line
```suggestion
        </h3>
        
        <p class="product-card__sku">Product code: 195434</p>
```